crates/nu-command/src/filesystem/ls.rs

```diff
@@ -980,9 +980,18 @@ mod windows_helper {
 fn read_dir(
     f: &Path,
```

Comment:
Still, I think `use_thread` parameter is required.  We don't want to sort items if it's using thread.